public/stylesheet.css

```diff
@@ -229,7 +229,7 @@ a {
   }
 }
 @media (max-width: 640px) {
-  .header {
+  x .header {
```

Comment:
This doens't look right @jorgeamadosoria

Reply:
fixed in #135